benchmarks/source-wordpress/package.json

```diff
@@ -23,12 +23,13 @@
     "gatsby-transformer-sharp": "^2.3.14",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
-    "ts-node": "^8.9.0",
     "typescript": "^3.8.3"
```

Comment:
Don't need this anymore.

Reply:
@duffn this diff is it being removed :p
This did prompt me to remove 2 other unused packages tho!

Reply:
Oops, did I tag the wrong line? I was actually trying to mention `typescript` but I'm glad it prompted that anyway!

Reply:
Oh 🤦 that makes more sense hahah. Thanks Duffy!